src/types.rs

```diff
@@ -554,15 +575,34 @@ impl From<Term> for Context {
 /// [`Operator`]: struct.Operator.html
```

Comment:
Please give some examples here constructing `Term`s a couple different ways (e.g. variables, applications, constants (i.e. applications of operators with arity 0), and a mix of these built from the parser or by hand.)

Reply:
Nice job!